Description: Fix scripts to expect translation files in the standard folder
Author: Nicolas Bourdaud <nicolas.bourdaud@gmail.com>
Origin: vendor
Forwarded: no
Applied-Upstream: no
Last-Update: 2013-05-16

--- a/files/generate_desktop_files
+++ b/files/generate_desktop_files
@@ -26,12 +26,12 @@
 """)
 
 import gettext
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 
-for directory in os.listdir("/usr/share/cinnamon/locale"):
-	if os.path.isdir(os.path.join("/usr/share/cinnamon/locale", directory)):
+for directory in os.listdir("/usr/share/locale"):
+	if os.path.isdir(os.path.join("/usr/share/locale", directory)):
 		try:
-			language = gettext.translation('cinnamon-settings', "/usr/share/cinnamon/locale", languages=[directory])
+			language = gettext.translation('cinnamon-settings', "/usr/share/locale", languages=[directory])
 			language.install()
 			desktopFile.writelines("Name[%s]=%s\n" % (directory, _(menuName)))
 		except:
@@ -39,10 +39,10 @@
 
 desktopFile.writelines("Comment=Fine-tune Cinnamon settings\n")
 
-for directory in os.listdir("/usr/share/cinnamon/locale"):
-	if os.path.isdir(os.path.join("/usr/share/cinnamon/locale", directory)):
+for directory in os.listdir("/usr/share/locale"):
+	if os.path.isdir(os.path.join("/usr/share/locale", directory)):
 		try:
-			language = gettext.translation('cinnamon-settings', "/usr/share/cinnamon/locale", languages=[directory])
+			language = gettext.translation('cinnamon-settings', "/usr/share/locale", languages=[directory])
 			language.install()			
 			desktopFile.writelines("Comment[%s]=%s\n" % (directory, _(menuComment)))
 		except:
--- a/files/usr/lib/cinnamon-settings/cinnamon-settings.py
+++ b/files/usr/lib/cinnamon-settings/cinnamon-settings.py
@@ -27,7 +27,7 @@
     sys.exit(1)
 
 # i18n
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 # i18n for menu item
 menuName = _("System Settings")
 menuComment = _("Control Center")
--- a/js/ui/main.js
+++ b/js/ui/main.js
@@ -149,7 +149,7 @@
 
 // Override Gettext localization
 const Gettext = imports.gettext;
-Gettext.bindtextdomain('cinnamon', '/usr/share/cinnamon/locale');
+Gettext.bindtextdomain('cinnamon', '/usr/share/locale');
 Gettext.textdomain('cinnamon');
 const _ = Gettext.gettext;
 
--- a/files/usr/share/cinnamon/applets/panel-launchers@cinnamon.org/cinnamon-add-panel-launcher.py
+++ b/files/usr/share/cinnamon/applets/panel-launchers@cinnamon.org/cinnamon-add-panel-launcher.py
@@ -11,7 +11,7 @@
 import sys
 import gettext
 
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 Settings = Gio.Settings.new("org.cinnamon")
 
 _ = gettext.gettext
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -80,7 +80,7 @@
 	$(CINNAMON_CFLAGS)			\
 	-I$(srcdir)/tray			\
 	-DVERSION=\"$(VERSION)\"		\
-	-DLOCALEDIR=\"$(datadir)/cinnamon/locale\" 	\
+	-DLOCALEDIR=\"$(datadir)/locale\" 	\
 	-DDATADIR=\"$(datadir)\"		\
 	-DCINNAMON_LIBEXECDIR=\"$(libexecdir)\"	\
 	-DCINNAMON_DATADIR=\"$(pkgdatadir)\"	\
--- a/files/usr/lib/cinnamon-screensaver-lock-dialog/cinnamon-screensaver-lock-dialog.py
+++ b/files/usr/lib/cinnamon-screensaver-lock-dialog/cinnamon-screensaver-lock-dialog.py
@@ -9,7 +9,7 @@
 import socket
 
 # i18n
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 
 class MainWindow:
   
--- a/files/usr/lib/cinnamon-settings/modules/cs_backgrounds.py
+++ b/files/usr/lib/cinnamon-settings/modules/cs_backgrounds.py
@@ -11,7 +11,7 @@
 import subprocess
 import tempfile
 
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 
 BACKGROUND_MODES = [
     ("wallpaper", _("Wallpaper")),
--- a/files/usr/lib/cinnamon-settings/modules/cs_keyboard.py
+++ b/files/usr/lib/cinnamon-settings/modules/cs_keyboard.py
@@ -5,7 +5,7 @@
 import cgi
 import gettext
 
-gettext.install("cinnamon", "/usr/share/cinnamon/locale")
+gettext.install("cinnamon", "/usr/share/locale")
 
 # Keybindings page - check if we need to store custom
 # keybindings to gsettings key as well as GConf (In Mint 14 this is changed)
